big-game/src/app/sign-up/sign-form/sign-form.component.html

```diff
@@ -0,0 +1,79 @@
+<div class="row">
+  <div class="col-md-8 ml-md-auto">
+    <form [formGroup]="signForm" (ngSubmit)="onSubmit()">
+      <div formGroupName="userData">
+        <div class="form-group">
+          <input formControlName="username" placeholder="User Name" type="text" class="form-control" />
+          <span *ngIf="
+              !signForm.get('userData.username').valid &&
+              signForm.get('userData.username').touched
+            " class="text-danger">Please enter a valid username!
+          </span>
+        </div>
+
```

Comment:
inconsistent code -2

Reply:
What part? - what does inconsistent mean? - how do we fix it/where? make this code consistent with what?